client/app/components/composites/ListingCard/ListingCard.story.js

```diff
@@ -113,6 +118,50 @@ const ListingCardImageError =
           value: 9, // eslint-disable-line no-magic-numbers
           unit: ':miles',
         }),
+        author: {
+          familyName: 'family name',
+          givenName: 'given name',
+          description: 'product author',
+          avatarURL: 'https://placehold.it/40x40',
+          profileURL: `#profile${Math.random(10)}`, // eslint-disable-line no-magic-numbers
+        },
+      }),
+    },
+  );
+
+const ListingCardNoPrice =
+  r(ListingCard,
+    {
+      className: css.listing,
+      color: '#347F9D',
+      listing: new ListingModel({
+        id: 'lkjg84573874yjdf',
+        title: 'Title',
+        images: new Immutable.List([new ImageRefs({
+          square: new Image({
+            url: 'https://placehold.it/408x408',
+          }),
+          square2x: new Image({
+            type: 'square2x',
+            width: 816,
+            height: 816,
+            url: 'https://placehold.it/816x816',
+          }),
+        })]),
+        listingURL: 'https://example.com/listing/342iu4',
+        orderType: new Immutable.Map({ en: 'Giving away', fi: 'Annetaan' }),
+        price: null,
+        distance: new Distance({
+          value: 12972, // eslint-disable-line no-magic-numbers
+          unit: ':miles',
+        }),
+        author: {
+          familyName: 'family name',
+          givenName: 'given name',
+          description: 'product author',
+          avatarURL: 'https://placehold.it/40x40',
+          profileURL: `#profile${Math.random(10)}`, // eslint-disable-line no-magic-numbers
+        },
```

Comment:
Also this: couldn't this class have a base version of listing card data, and alter it to match the cases?